app/controllers/articles_controller.rb

```diff
@@ -104,7 +104,8 @@ def preview
             title: parsed["title"],
             tags: (Article.new.tag_list.add(parsed["tags"], parser: ActsAsTaggableOn::TagParser) if parsed["tags"]),
             cover_image: (ApplicationController.helpers.cloud_cover_url(parsed["cover_image"]) if parsed["cover_image"])
-          }
+          },
+                 status: :ok
```

Comment:
looks like the formatting has gone a little bit awry but if the linters are happy... 🙂